### ISSUE TITLE
Edit button has a space between the project URL and the suffix.

### ISSUE THREAD

**Issue by sritchie** [47deg/sbt-microsites#323]
Hey all,

When I include the following in my config (using sbt-microsites 0.9.3):

```scala
 micrositeEditButton := Some(
   MicrositeEditButton(
     "Help us improve this page",
     "/edit/develop/docs/src/main/tut/{{ page.path }}")),
```

I see the following button HTML generated:

```html
<a href="https://github.com/sritchie/scala-rl /edit/develop/docs/src/main/tut/course/tabular/bandits.md" class="btn-sm btn-info">Improve this Page</a>
```

Happy to submit a PR to fix, but just wanted to submit to see if you all had any thoughts.

Cheers,
Sam

**Comment by juanpedromoreno** [comment]
Thanks for reporting this issue @sritchie . Personally, I didn't see this issue before, I've tried to look at your issue cloning your repo but I can't resolve all the dependencies (rainier-cats).

The double `href` [that you already saw](https://github.com/47deg/sbt-microsites/pull/278#issuecomment-526990154) is definitely suspicious.

**Comment by calvellido** [comment]
After having a quick look, this seems to only be happening in the 0.9.3 version of `sbt-microsites`, which beyond some doc changes updated some dependencies:

https://github.com/47deg/sbt-microsites/releases

Not sure yet about the actual culplrit of this, but for a quick fix maybe you can use 0.9.2 in your project @sritchie, or if you think you have the solution for it, please go ahead!

**Comment by juanpedromoreno** [comment]
Probably this upgrade might be the culprit? https://github.com/47deg/sbt-microsites/pull/321/files#diff-4198f89a479a3a8c2b310affc814c14fR26

**Comment by calvellido** [comment]
Apparently only [Scala 2.13.0 support was added in `scala-tags` 0.7.0](https://github.com/lihaoyi/scalatags/releases):

https://github.com/lihaoyi/scalatags/commit/28e1b1e0027def1e121b51a40dc746807e281540

:roll_eyes: 

 but yeah... It seems like that, and the double `href` could be messing the URL formation.

**Comment by sritchie** [comment]
Hey @calvellido, thanks for the quick fix! @juanpedromoreno, yeah, I'm waiting for the Rainier team to publish a develop snapshot with some code I pushed upstream... I was fried last night and couldn't figure out an easy way to push a jar under `io.samritchie` that would get everything resolving.

By the way, the 0.9.2 suggestion worked great: https://www.scalarl.com/course.html

This project is unusably early in development, but I was so excited to get a microsite up that I made it public anyway :)

Thank you both!
